src/config.ts

```diff
@@ -63,6 +63,13 @@ export default {
         title: 'bsb',
         description: 'The path to the `bsb` binary.',
         default: 'bsb',
+        order: 20,
+      },
+      bsc: {
+        type: 'string',
+        title: 'bsc',
+        description: 'The path to the `bsc` binary.',
+        default: 'bsc',
```

Comment:
I decided to place it next to `bsb` in config, tho it caused some inconvenience w/ types (OSL config doesn't have `bsc` property).